index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="https://kit.fontawesome.com/722afd38e8.js" crossorigin="anonymous"></script>
+</head>
+<body>
+   
+  <header class="header">
+    <a href="index.html" class="my-logo"><span>My Logo</span></a>
+
+    <nav class="dropdown">
+        <button class="dropbtn"><img src="assets/images/button.png" alt=""></button>
+        <div class="dropdown-content">
+          <a href="#">Portfolio</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </div>
+    </nav>
+
+  </header>
+   
+    <section class="headline">
+      
+        <h2>I’m Ahmed Glad to see you!</h2>
+
+        <p>I’m a software developer! I can help you build a product,
+             feature or website Look through some of my work and experience! 
+             If you like what you see and have a project you need coded,
+             don’t hestiate to contact me.</p>  
+
+            <div class="Social"> 
+                <span class="Text"><h2>LET’S CONNECT</h2></span>
+             </div>
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you always use some semantic tags. `heading` element inside a `span` inside a `div` in this context doesn't have any semantic meaning. Instead of all of that You may consider only a heading element `h2`